### PR TITLE
Add support for Orange PI via WiringOP

### PR DESCRIPTION
**Pull request by vojtechkral**
Besides Raspberry PI I also have an Orange PI board, which requires patched version of WiringPI. Feel free to merge if you find this ok. Thanks!

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,5 +5,11 @@ wiringpi:
...
+wiringop:
+	@echo >&2 $(CFLAGS)
+	$(MAKE) -C WiringOP/wiringPi clean
+	$(MAKE) -C WiringOP/wiringPi static CC=arm-linux-gnueabihf-gcc DEBUG=-O2
+	rm -f $(OUT_DIR)/libwiringpi.a
+	cp WiringOP/wiringPi/libwiringPi.a $(OUT_DIR)/libwiringpi.a
 
-.PHONY: wiringpi
\ No newline at end of file
+.PHONY: wiringpi
```
Comment (Ogeon) [inline Makefile]: `wiringop` should probably also be marked as `.PHONY`.

**Comment by Ogeon** [comment]
Nice! I see no reason why not :smile: Would you mind mentioning the existence of the Cargo feature in the readme?

**Comment by vojtechkral** [comment]
Ok, it's updated.

By the way, I wanted to ask, how do you do developement builds? Because, obviously, the setup functions will not work on a regular computer, they need the BCM hardware to function.

**Comment by Ogeon** [comment]
Looks good!

I'm usually cross-compiling and testing it in a hobby project. This worked fine when I was the only developer. It's not as easy with situations like this, but I suppose I can trust this one to work, since the API should be the same for both versions.

It would be nice to have a more reliable testing method, though. Something that can be automated on Travis.

**Comment by Ogeon** [comment]
Looks like this builds as expected, so I'll merge it. Thank you!

**Comment by vojtechkral** [comment]
Ok, thanks!

**Comment by Ogeon** [comment]
I found a way to make build tests on Travis, so future PRs should now be less error prone.

**Pull request merged**